ui/hooks/gasFeeInput/useGasFeeErrors.js

```diff
@@ -262,5 +262,6 @@ export function useGasFeeErrors({
     gasWarnings,
     balanceError,
     estimatesUnavailableWarning,
+    hasSimulationError: Boolean(transaction?.simulationFails),
```

Comment:
perhaps I'm not simulating correctly, but if I set this to true, I'm still seeing the confirm button enabled without acknowledging:
<img width="441" alt="Screen Shot 2021-11-24 at 10 18 53 AM" src="https://user-images.githubusercontent.com/34557516/143275732-d25ca00f-b471-47a9-9175-1d1986d69d47.png">

is that the desired behavior?

Reply:
confirm button uses transaction directly, it is just issue with simulating the simulation error